cleanup.go

```diff
@@ -11,7 +11,9 @@ var defaultInterval = time.Minute * 5
 // sessions from the database.
 //
 // The design is based on https://github.com/yosssi/boltstore
-func (db *PGStore) Cleanup(interval time.Duration) (chan<- struct{}, <-chan struct{}) {
+func (db *PGStore) Cleanup(
```

Comment:
It's not worth splitting function declarations across lines for the sake of 80 cols. The standard lib doesn't do this.